Src/WitsmlExplorer.Api/HttpHandlers/LogHandler.cs

```diff
@@ -40,8 +41,15 @@ public static async Task<IResult> GetLogCurveInfo(string wellUid, string wellbor
         {
             if (mnemonics.Any())
             {
-                LogData logData = await logObjectService.ReadLogData(wellUid, wellboreUid, logUid, mnemonics.ToList(), startIndexIsInclusive, startIndex, endIndex);
-                return Results.Ok(logData);
+                try
+                {
+                    LogData logData = await logObjectService.ReadLogData(wellUid, wellboreUid, logUid, mnemonics.ToList(), startIndexIsInclusive, startIndex, endIndex);
+                    return Results.Ok(logData);
+                }
+                catch (WitsmlException e)
+                {
+                    return Results.UnprocessableEntity(new ErrorDetails() { Message = e.Message, StatusCode = 422 });
```

Comment:
For a little better readability you can if you want use `StatusCodes.Status422UnprocessableEntity` from `Microsoft.AspNetCore.Http`

Reply:
I agree that 422 client error is a little bit misguiding, but the alternative I guess will be 500